File: Coding/java Programs/fact.java
import java.util.Scanner;
public class fact{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();   
        int fact = 1;
        if(n>1){
            for(int i = 1; i <= n; i++){
                fact *= i;  
            }
            System.out.println("Factorial of " + n + " is: " + fact);
        } else if (n == 0 || n == 1) {
            System.out.println("Factorial of " + n + " is: 1");
        } else {
            System.out.println("Factorial is not defined for negative numbers.");
        }
    }
}

File: Coding/Python programs/fact.py
n=int(input("Enter a number: "))
def factorial(n):
    global fact
    if n == 0 or n == 1:
        return 1
    else:
        fact = n * factorial(n - 1)
        return fact 

factorial(n)
print(f"The factorial of {n} is: {fact}")   
        